Automattic/jetpack
# Jetpack: remove old widget migration code

## Proposed changes:
Remove old migration code for legacy Gallery and Image widgets.

### Other information:


## Jetpack product discussion
https://github.com/Automattic/vulcan/issues/529
p9dueE-56-p2
p1730913512335599-slack-C05PV073SG3

## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:
1. Make sure the tests pass.
2. Try using one of the widgets from the [Widget module](https://github.com/Automattic/jetpack/tree/trunk/projects/plugins/jetpack/modules/widgets) in a post, confirm it renders well.
